### PR TITLE
CI: Add GitHub workflows

### PR DESCRIPTION
**Pull request by ohadTsarfati**
* added github workflow ci process
* removed travis yaml

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        with:
+          node-version: 14
+      - name: lint package-lock
+        run: npx lockfile-lint --path package-lock.json --allowed-hosts npm --validate-https
+
+  test:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+        node: ['8', '10', '12']
```
Comment (MatanAvneri) [inline .github/workflows/ci.yaml:21]: You can upgrade the node versions to 10, 12 and 14

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install Dependencies
+        run: npm ci
+      - name: Run Tests & Collect Coverage
+        run: npm run test:coveralls
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        if: matrix.node == '12'
```
Comment (MatanAvneri) [inline .github/workflows/ci.yaml:36]: if the version will be 14 you can change this one as well

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (ohadTsarfati) [reply .github/workflows/ci.yaml:21]: doesn't it require a major version update?

Comment (MatanAvneri) [reply .github/workflows/ci.yaml:21]: Right, saw the readme... So just add `14`, without removing the rest.

**Review by MatanAvneri — APPROVED** [review]

**Pull request merged**